### ISSUE TITLE
where is Self-Supervised Contrastive Loss?

### ISSUE THREAD

**Issue by seyeeet** [HobbitLong/SupContrast#85]
thanks for the code,
can you guide me where the Self-Supervised Contrastive Loss is defined in the code, in the paper it is defined at [eq 1](https://arxiv.org/pdf/2004.11362.pdf)
to be clear I see that [here ](https://github.com/HobbitLong/SupContrast/blob/8d0963a7dbb1cd28accb067f5144d61f18a77588/losses.py#:~:text=mask%20%3D%20mask%20*%20logits_mask-,%23%20compute%20log_prob,-exp_logits%20%3D%20torch.exp) we have the loss 
but this 
```
# compute log_prob
        exp_logits = torch.exp(logits) * logits_mask
        log_prob = logits - torch.log(exp_logits.sum(1, keepdim=True))

        # compute mean of log-likelihood over positive
        mean_log_prob_pos = (mask * log_prob).sum(1) / mask.sum(1)

        # loss
        loss = - (self.temperature / self.base_temperature) * mean_log_prob_pos
        loss = loss.view(anchor_count, batch_size).mean()

```
is different from eq1 in my opinion, isnot it?
